deliver/lib/deliver/upload_screenshots.rb

```diff
@@ -91,7 +99,9 @@ def collect_screenshots_for_languages(path)
 
         UI.important("Framed screenshots are detected! 🖼 Non-framed screenshot files may be skipped. 🏃") if prefer_framed
 
-        language = File.basename(lng_folder)
+        language_dir_name = File.basename(lng_folder)
+        language = available_languages[language_dir_name.downcase] || language_dir_name
```

Comment:
instead of using `language_dir_name` if `available_languages[language_dir_name.downcase]` returns `nil`, would it make sense to raise an error here that you aren't using a supported language?